bpf/include/iproute2/bpf_elf.h

```diff
@@ -35,6 +35,10 @@ struct bpf_elf_map {
 	__u32 flags;
 	__u32 id;
 	__u32 pinning;
+#ifdef SOCKMAP
+	__u32 inner_id;
+	__u32 inner_idx;
+#endif
```

Comment:
Is this used somewhere?

Reply:
On the golang side where the map is created these fields are needed for hash of maps. I guess these could be removed here and things would continue to work. OTOH those fields are part of the maps description.

Reply:
I was considering unconditionally enabling them but wasn't sure if all iproute2 versions could handle extra fields like that.